dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandler.kt

```diff
@@ -44,6 +45,10 @@ internal class DatadogLogHandler(
             val log = createLog(level, message, throwable, attributes, tags, resolvedTimeStamp)
             writer.write(log)
         }
+
+        if (level >= AndroidLog.ERROR) {
```

Comment:
hmmm....don't we also sample this, or are we going to use the RUM sampling mechanism ?

Reply:
Since those are becoming RUM Errors, we don't want to sample them with the Logs sampling rule. The RumMonitor will perform the sampling directly